CHANGES.rst

```diff
@@ -12,6 +12,7 @@ Unreleased
 - Added ``tracked_fields`` attribute to historical models (gh-1038)
 - Fixed ``KeyError`` when running ``clean_duplicate_history`` on models with ``excluded_fields`` (gh-1038)
 - Added support for Python 3.11 (gh-1053)
+- Fix missing comma in the `Signals` par of the docs
```

Comment:
It's perhaps not necessary to add this line, as the changes in this PR can be considered described by the "Fixed typos in the docs" line above 🙂 
```suggestion
```

Reply:
The mentioned "Fixed typos in the docs" line is not part of the "Unreleased" section anymore, so I copied it there :)